eng/build.ps1

```diff
@@ -752,7 +771,7 @@ try {
   catch
   {
     if ($ci) {
-      echo "##vso[task.logissue type=error](NETCORE_ENGINEERING_TELEMETRY=Test) Tests failed"
+      Write-LogIssue -Type "error" -Message "(NETCORE_ENGINEERING_TELEMETRY=Build) Tests failed"
```

Comment:
Why the telemetry change?

Reply:
To my understanding we should always be using `Write-LogIssue` to emit the `##vsa[task.logisuse` commands. It's meant ot be a strongly typed wrapper we can query for and customize.

Reply:
I was asking why `NETCORE_ENGINEERING_TELEMETRY=Test` changed to `NETCORE_ENGINEERING_TELEMETRY=Build`.

Reply:
copy / paste stupidity 😦